dev/ci/ci-basic-overlay.sh

```diff
@@ -79,8 +79,8 @@
 ########################################################################
 # VST
 ########################################################################
-: ${VST_CI_BRANCH:=less_init_plugins}
-: ${VST_CI_GITURL:=https://github.com/letouzey/VST.git}
+: ${VST_CI_BRANCH:=master}
+: ${VST_CI_GITURL:=https://github.com/Zimmi48/VST.git}
```

Comment:
The reason is the same as why it wasn't PrincetonUniversity before: there is an additional commit for compatibility with the "Less Init Plugins" version. The problem is not just with this particular development and there is an open bug report about that. It should get fixed after the 8.6.1 release which includes dummy plugins for improved backward compatibility.